---
geometry_msgs/PoseArray people_locations
